internal/envvar/envvar.go

```diff
@@ -1,3 +1,34 @@
+/*
+   Copyright 2024 Docker Terraform Provider authors
```

Comment:
copyright blurbs seem to be duplicated in each file 😅

Reply:
:grimacing: not sure how that happened. fixed!